homeassistant/components/device_tracker/__init__.py

```diff
@@ -96,6 +99,19 @@
     vol.Optional(CONF_NEW_DEVICE_DEFAULTS,
                  default={}): NEW_DEVICE_DEFAULTS_SCHEMA
 })
+SERVICE_SEE_PAYLOAD_SCHEMA = vol.Schema(vol.All(
+    cv.has_at_least_one_key(ATTR_MAC, ATTR_DEV_ID), {
+        ATTR_MAC: cv.string,
+        ATTR_DEV_ID: cv.string,
+        ATTR_HOST_NAME: cv.string,
+        ATTR_LOCATION_NAME: cv.string,
+        ATTR_GPS: cv.gps,
+        ATTR_GPS_ACCURACY: cv.positive_int,
+        ATTR_BATTERY: cv.string,
```

Comment:
Should this be a string or an int?

Reply:
Both should be possible, as it turns out. Some platforms use int, others str.

Reply:
Ok, well a test failed due to it expecting an int, it seems. If we coerce to `int` platforms should be able to provide both, I think.